examples/with-docker-compose/next-app/src/pages/index.tsx

```diff
@@ -0,0 +1,91 @@
+import Head from 'next/head'
+import Image from 'next/image'
+import { GetServerSideProps } from 'next'
+import styles from '../styles/Home.module.css'
+
+type Props = {
+  serverOnlyVariable: string
+}
+
+export const getServerSideProps: GetServerSideProps<Props> = async () => {
+  return {
+    props: { serverOnlyVariable: process.env.ENV_VARIABLE },
```

Comment:
I wonder if this might give the wrong impression to developers about how to use environment variables and some might accidentally expose secrets this way. What do you think?

Reply:
Good point! I've removed this, since there's no need to showcase a basic feature of Next.js.